src/ai/backend/storage/api/manager.py

```diff
@@ -776,14 +778,32 @@ class Params(TypedDict):
     ) as params:
         await log_manager_api_entry(log, "mkdir", params)
         ctx: RootContext = request.app["ctx"]
+        vfid = params["vfid"]
+        parents = params["parents"]
+        exist_ok = params["exist_ok"]
+        relpath = params["relpath"]
+        relpaths = relpath if isinstance(relpath, list) else [relpath]
+
         async with ctx.get_volume(params["volume"]) as volume:
-            with handle_fs_errors(volume, params["vfid"]):
-                await volume.mkdir(
-                    params["vfid"],
-                    params["relpath"],
-                    parents=params["parents"],
-                    exist_ok=params["exist_ok"],
-                )
+            with handle_fs_errors(volume, vfid):
```

Comment:
we do not need `handle_fs_errors()` here because it cannot catches any errors since we handle error as a result of tasks of `gather()`